ext/opentelemetry-ext-wsgi/src/opentelemetry/ext/wsgi/__init__.py

```diff
@@ -105,17 +138,20 @@ def __call__(self, environ, start_response):
             start_response: The WSGI start_response callable.
         """
 
+        start_timestamp = time_ns()
+
         tracer = trace.tracer()
-        path_info = environ["PATH_INFO"] or "/"
-        parent_span = propagators.extract(_get_header_from_environ, environ)
+        parent_span = propagators.extract(get_header_from_environ, environ)
+        span_name = get_default_span_name(environ)
 
         span = tracer.create_span(
-            path_info, parent_span, kind=trace.SpanKind.SERVER
+            span_name, parent_span, kind=trace.SpanKind.SERVER
         )
-        span.start()
+        span.start(start_timestamp)
```

Comment:
The code above conflicts with the changes to remove create_span from the API,  https://github.com/open-telemetry/opentelemetry-python/issues/152. Just making a note here that `start_span` doesn't allow for a start timestamp to be passed in, would be nice to have for cases like this, likely this could be changed in PR #290

Reply:
I haven't reviewed #290 but IMHO it must keep the capability of adding a start time. This is both in the spec and was explicitly requested, see #134.

Reply:
👍